examples/policies/host/lock-down-ingress.yaml

```diff
@@ -6,17 +6,17 @@ metadata:
 spec:
   nodeSelector:
     matchLabels:
-      type: worker
+      type: ingress-worker
   ingress:
   - toPorts:
     - ports:
-      - port: "6443"
+      - port: "KUBE_API_SERVER"
         protocol: TCP
-      - port: "22"
+      - port: "ACCESS_PORTS_INTO_NODES"
         protocol: TCP
-      - port: "2379"
+      - port: "ETCD"
         protocol: TCP
-      - port: "4240"
+      - port: "HEALTH_CHECKS"
         protocol: TCP
-      - port: "8472"
+      - port: "VXLAN"
```

Comment:
Why the removal of the exact port numbers? 99% of the users won't need to change these values, the other 1% will know which values will need to be modified to suit their needs.

Reply:
@aanm the goal is to mitigate against the possibility of an operator copying this configuration directly during testing and breaking Kube in their environment. I chatted with @qmonnet about how we can mitigate against this and we both thought a good start is preventing an example policy from loading unless the user changes example values. I'm happy to discuss alternative solutions.

Reply:
s/ @qmonnet / @pchaigno /

Reply:
@jedsalazar I understand that, but 99% of the environments are using those exact port numbers, right? So what I am saying is that 100% of our users will have to research and add those ports number manually for which 99% of them would work for those defaults. Maybe we could add this port:
```
      - port: "REMOVE_ME_AFTER_DOUBLE_CHECKING_PORTS"
        protocol: UDP
```
or something like that so it would force the user to at least double check the ports installed.

Reply:
What I had in mind was simply replacing the SSH port number with a string, so that users have to pause and double check that config. I think other numbers should stay since they are needed anyway. Adding a new entry to force users to double check, as André suggested, is also an option :+1: 

Another option we discussed with Jed was to simply allow all from outside world in this example policy. We would still have to be careful about ports needed for kube-api, etc. of course.

Reply:
Change has been applied.